### DISCUSSION TITLE
Site Thumbnail

### DISCUSSION THREAD

**Discussion by mkz71** [themefisher/airspace-hugo#114] [Q&A]
How can I add site thumbnail?

**Comment by salim-b** [comment]
What do you mean by "thumbnail"? [Favicons](https://en.wikipedia.org/wiki/Favicon)? To create Favicons, I'd recommend to feed your site logo to the [RealFaviconGenerator](https://realfavicongenerator.net/) and then simply put the generated Favicons under `static/`.

**Reply by mkz71** [reply]
No, I am talking about when you send url to someone and it will show link preview photo. How can I make it to show specific photo?

**Reply by salim-b** [reply]
Ok, so you're talking about [Open Graph](https://ogp.me/) and/or [Twitter Card]() metadata. This is actually already implemented in the theme, it uses Hugo's internal templates for that. You'll find documentation about it [here](https://gohugo.io/templates/internal/#open-graph) and [here](https://gohugo.io/templates/internal/#twitter-cards).

tl;dr: You can set an image individually for every page/post by setting ~~`images:`~~ `image:` in the front matter as well as a global fallback.

Fallback is set in `config.toml`:
https://github.com/themefisher/airspace-hugo/blob/3d086e74e22824afc858e96237ff5b5f5b8499ec/exampleSite/config.toml#L45

**Reply by phil-pappas** [reply]
> What do you mean by "thumbnail"? [Favicons](https://en.wikipedia.org/wiki/Favicon)? To create Favicons, I'd recommend to feed your site logo to the [RealFaviconGenerator](https://realfavicongenerator.net/) and then simply put the generated Favicons under `static/`.

Hi Salim. I placed a favicon.png (from the generator you indicated) in the `images/` folder but it doesn't appear on the tab.

**Reply by salim-b** [reply]
That's probably because you need to place the favicon under **`static/`**...

(Note that in general favicon changes are a bit tricky to test because browsers tend to cache them for very long periods.)
